src/pages/_app.js

```diff
@@ -1,14 +1,66 @@
 import React from "react";
 import CyverseAppBar from "../components/appBar/CyVerseAppBar";
+import NavigationConstants from "../components/navigation/NavigationConstants";
 import theme from "../components/theme/default";
 import { ThemeProvider } from "@material-ui/core/styles";
+import getConfig from "next/config";
+import { useRouter } from "next/router";
+
+function MyApp({ Component, pageProps, intercomAppId, intercomEnabled }) {
+    const router = useRouter();
+    const pathname = router.pathname
+        ? router.pathname.slice(1)
+        : NavigationConstants.DASHBOARD;
 
-function MyApp({ Component, pageProps }) {
     React.useEffect(() => {
         const jssStyles = document.querySelector("#jss-server-side");
         if (jssStyles) {
             jssStyles.parentElement.removeChild(jssStyles);
         }
+        const APP_ID = intercomAppId;
+        const enabled = intercomEnabled;
```

Comment:
Minor, but could we not just use `intercomAppId` and `intercomEnabled` directly?